src/site/content/en/blog/designcember-calculator/index.md

```diff
@@ -0,0 +1,300 @@
+---
+layout: post
+title: 'Designcember Calculator'
+authors:
+  - thomassteiner
+hero: image/kheDArv5csY6rvQUJDbWRscckLr1/kKhOZkehYIQ49GQOXrsw.png
+alt: 'Designcember'
+thumbnail: image/kheDArv5csY6rvQUJDbWRscckLr1/qRLa7xFsolDUbEfyugoW.png
+subhead: >
+  A skeuomorphic attempt at recreating a solar calculator on the web with the Window Controls
+  Overlay API and the Ambient Light Sensor API.
+description: >
+  A skeuomorphic attempt at recreating a solar calculator on the web that makes use of the ambient
+  light sensor and the window controls overlay feature.
+date: 2021-12-26
+tags:
+  - blog
+  - capabilities
+  - css
+---
+
+## The challenge
+
+I am a kid of the 1980s. A thing that was all the rage back when I was in highschool were solar
+calculators. We were all given a
+[TI-30X SOLAR](https://en.wikipedia.org/wiki/TI-30#/media/File:TI-30X_SOLAR,_2.jpg) by the school,
+and I have fond memories of us benchmarking our calculators against each other by calculating the
+factorial of 69, the highest number the TI-30X could handle. (The speed variance was very
+measurable, I have still no idea why.)
```

Comment:
Manufacturing deficiencies? [I've read that](https://www.amazon.com/Dealers-Lightning-Xerox-PARC-Computer/dp/0887309895/ref=sr_1_1?crid=QALGZ279BM9J&keywords=dealers+in+lightning&qid=1639072837&sprefix=dealers+in+light%2Caps%2C201&sr=8-1) the first line of integrated circuits that Intel produced were so bad coming off the production line that they had college educated engineers testing every unit that came off the line. The guys who did it called it 'turd polishing'.

Reply:
It was several seconds at times. We all got the same calculators at the same time, I guess manufacturing variances, too, but it astonishes me…